compiler/one-cmds/onelib/CfgRunner.py

```diff
@@ -109,6 +114,8 @@ def run(self, working_dir, verbose=False):
                 options += ['-O', self.opt]
             if verbose:
                 options.append('--verbose')
+            if section == 'one-codegen' and self.backend:
```

Comment:
Not for this PR, I think it would be better to split CfgRunner (maybe implement a different version for each driver), because it is doing too many jobs..